yt/frontends/rockstar/data_structures.py

```diff
@@ -20,7 +20,7 @@ class RockstarBinaryFile(HaloCatalogFile):
     header: dict
     _position_offset: int
     _member_offset: int
-    _Npart: np.array
+    _Npart: np.ndarray[Any, np.dtype[np.int64]]
```

Comment:
you'll still need to quote this, because `np.ndarray` only became subscriptable in numpy 1.22, which is more recent than our current minimal requirement